### DISCUSSION TITLE
Same component selection page like the MVC Core project

### DISCUSSION THREAD

**Discussion by weedkiller** [Dotnet-Boxed/Templates#132] [Q&A]
Hi can you please add the same options page like the MVC core project, in the MVC 4.5/5 its missing
Also you have a lot of components via DI, how do we turn them off.
Can you add recaptcha and throttling
Also the latest nuget does not reflect the changes from the push this month, is there an update coming

thanks

**Comment by RehanSaeed** [comment]
Lots of questions:
- No plans to add feature selection to the MVC 5 project. I'm only one person and don't have time!
- In MVC 5 you can just delete stuff you don't need.
- No plans for adding a captcha. There is nothing to add it to.
- There is always an update coming.
